test/libsolidity/SemanticTest.cpp

```diff
@@ -508,17 +508,27 @@ TestCase::TestResult SemanticTest::runTest(
 			_stream << errorReporter.format(_linePrefix, _formatted);
 		}
 		_stream << endl;
+
 		AnsiColorized(_stream, _formatted, {BOLD, CYAN}) << _linePrefix << "Obtained result:" << endl;
+
 		for (TestFunctionCall const& test: m_tests)
 		{
 			ErrorReporter errorReporter;
-			_stream << test.format(
+			std::string outputString =
+			test.format(
 				errorReporter,
 				_linePrefix,
 				m_gasCostFailure ? TestFunctionCall::RenderMode::ExpectedValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
 				_formatted,
-				/* _interactivePrint */ true
-			) << endl;
+				true
+			);
+
+			auto pos = outputString.find("  // ~");
+			while ((pos=outputString.find("  // ~", pos+6)) != std::string::npos) {
```

Comment:
1. The formatting is a bit off - please use spaces between operators, i.e. `pos = outputString...` and `pos + 6`.
2. Have you maybe thought about using something from Boost? `boost::replace_all` seems to be a good candidate, although I haven't taken a detailed look.

Reply:
Also, `std::` here is superfluous. We use `using namespace std` in cpp files.

Reply:
I have added the additional spacing. This replaces all but the first instance of the pattern. I would assume boost::replace_all would replace all of them. I don't know of anything in boost that would do the same thing but if there is I am happy to change it.